codec-framework/src/main/java/org/interledger/encoding/asn/codecs/AsnCharStringBasedObjectCodec.java

```diff
@@ -0,0 +1,141 @@
+package org.interledger.encoding.asn.codecs;
```

Comment:
The codecs should be neutral of serialization rules. They are basically an intermediary state for your data that resembles the ASN.1 form.

In theory you should be able to write serializers for DER too.

> Also, it would be cool to apply this codec-framework to the crypto-conditions code, which uses DER

This would be a good way to test the theory 😄